src/IdentityServer4/host/Views/Account/Login.cshtml

```diff
@@ -1,80 +1,76 @@
 @model LoginViewModel
 
 <div class="login-page">
-    <div class="page-header">
-        <h1>Login</h1>
-    </div>
-
-    <partial name="_ValidationSummary" />
+	<div class="page-header">
```

Comment:
It looks like you are using tabs for indenting here. Should probably be changed back to spaces.

Reply:
I can see that it is an issue in more than this file.

Reply:
ah. the old Tabs vs Spaces debate. I'll get those changed over to spaces and push the changes up today.